File: examples/python/hello.py
# -*- coding: utf-8 -*-

from field_reports import Bridge, ReportsError

param = {
    "template": {"paper": "A4"},
    "context": {
        "hello": {
            "new": "Tx",
            "value": "Hello, World!",
            "rect": [100, 700, 400, 750]
        }
    }
}
reports = Bridge.create_proxy()
pdf = reports.render(param)
with open("out.pdf", "wb") as f:
    f.write(pdf)

File: examples/ruby/hello.rb
# coding: utf-8

require "field_reports"

$param = {
    "template": {"paper": "A4"},
    "context": {
        "hello": {
            "new": "Tx",
            "value": "Hello, World!",
            "rect": [100, 700, 400, 750]
        }
    }
}

if ARGV.length == 1 then
    reports = FieldReports::Bridge.create_proxy()
    pdf = reports.render($param)
    File.binwrite(ARGV[0], pdf)
else
    p "usage: %s <outfile>" % $0
end

